src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -632,6 +632,11 @@ export default (accessToken, userID, opts) => {
       ({ partnerId, inquiryId }) =>
         `partner/${partnerId}/inquiry_request/${inquiryId}`
     ),
+    partnerLocationsConnectionLoader: gravityLoader(
```

Comment:
Just `partnerLocationsLoader` is probably sufficient (no need for 'connection' in the loader name)

Reply:
This loader was already in the unauthenticated gravity section, so brought it over here to get additional data on the CMS side.